package.json

```diff
@@ -6,7 +6,7 @@
   "types": "build/types/src/index.d.ts",
   "module": "build/index.mjs",
   "repository": "git@github.com:fibricheck/javascript-sdk.git",
-  "author": "jens.verbeken@craftzing.com",
+  "author": "Qompiumm",
```

Comment:
I think it's Qompium instead of Qompiummmmmmmmm ;) 
A glitch in your 🦋  keyboard? ;)

Reply:
damm 🙈 🐛

Reply:
good catch!